### PR TITLE
testnet & unit test mocks

### PR DESCRIPTION
**Pull request by brkomir**
Added mocks for scUSDC and scWETH and testnet deployment script. 
Let me know if you see some improvements that can be done, maybe an option to set ETH/WETH price could be helpful

### CODE REVIEW

File: test/mock/MockUSDC.sol [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.13;
+
+import {MockERC20} from "./MockERC20.sol";
```
Comment (coolhill) [inline test/mock/MockUSDC.sol:4]:
```suggestion
import {MockERC20} from "solmate/test/utils/mocks/MockERC20.sol";
```

File: test/scUSDC.unit.t.sol [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+import {IPool} from "aave-v3/interfaces/IPool.sol";
+import {IAToken} from "aave-v3/interfaces/IAToken.sol";
+import {IPoolDataProvider} from "aave-v3/interfaces/IPoolDataProvider.sol";
+
+import {scUSDC} from "../src/steth/scUSDC.sol";
+import {ISwapRouter} from "../src/interfaces/uniswap/ISwapRouter.sol";
+import {AggregatorV3Interface} from "../src/interfaces/chainlink/AggregatorV3Interface.sol";
+import {IVault} from "../src/interfaces/balancer/IVault.sol";
+
+import {MockERC4626} from "./mock/MockERC4626.sol";
```
Comment (coolhill) [inline test/scUSDC.unit.t.sol:19]:
```suggestion
import {MockERC4626} from "solmate/test/utils/mocks/MockERC4626.sol";```

Comment (brkomir) [reply test/scUSDC.unit.t.sol]: addressed in [this](https://github.com/lindy-labs/sandclock-contracts/pull/56/commits/2a6e386c830954963fb98ddd98e1ab110a8b2b35) commit

File: test/mock/MockUSDC.sol (hunk 1 above)
Comment (brkomir) [reply test/mock/MockUSDC.sol]: addressed in [this](https://github.com/lindy-labs/sandclock-contracts/pull/56/commits/2a6e386c830954963fb98ddd98e1ab110a8b2b35) commit

**Review by coolhill — APPROVED** [review]

**Pull request merged**